source/nvwave.py

```diff
@@ -122,9 +122,10 @@ def playWaveFile(
 
 	def play():
 		global fileWavePlayer
+		p = fileWavePlayer
```

Comment:
could you add a comment here to explain the reassignment? to make sure it doesn't get refactored out in future.